### ISSUE TITLE
What does the fine-tune mean here?

### ISSUE THREAD

**Issue by szubing** [wetliu/energy_ood#18]
what does the pre-train model from? train from what data?

**Comment by weitliu** [comment]
The pertained weights come from outlier-exposure. They are trained with Cifar10 or Cifar100.

**Comment by szubing** [comment]
> The pertained weights come from outlier-exposure. They are trained with Cifar10 or Cifar100.

I still don't understand. for example, if using Cifar10 as in data and SVHN as OOD test data, it seems the outlier-exposure data is from 80 Million Tiny Images. So the pretrained weights is from this 80 Million Tiny Images? but seems their are not groundtruths for this Tiny images dataset, how the pretrain work?

**Comment by YixuanLi** [comment]
Pre-trained weights refer to the in-distribution dataset. So that'd be CIFAR-10 weights trained with cross-entropy loss as initialization.

**Comment by szubing** [comment]
> 

I got it. Thanks
